tests/regression_tests/example_routes/smoothed/fuel/checkerboard.json

```diff
@@ -132,7 +132,9 @@
             "smoothing_max_iterations": 2000,
             "smoothing_blocked_sic": 10,
             "smoothing_merge_separation": 0.01,
-            "smoothing_converged_sep": 0.01
+            "smoothing_converged_sep": 0.01,
+            "adjust_waypoints": true,
```

Comment:
'adjust waypoints' and 'unit_shipspeed' is auto allocated so don't need to add the parameters here

Reply:
The config is saved as part of the output after everything is run, so will contain all the default values when they have been set. This is not strictly necessary but is how we've been doing things so far. Any new parameters are there because they were added during the refactor and this is the first time we've updated these test files since they were added, as long as they have the default value then they shouldn't have changed any results.